astropy/coordinates/angles.py

```diff
@@ -350,11 +350,17 @@ def plain_unit_format(val):
                 "The unit value provided is not an angular unit.")
 
         def do_format(val):
-            s = func(float(val))
-            if alwayssign and not s.startswith('-'):
-                s = '+' + s
-            if format == 'latex':
-                s = f'${s}$'
+            #Check if value is finite to avoid ValueErrors when turning it into
+            #a hexagesimal string.
```

Comment:
Nitpick, there should be a space.

```suggestion
            # Check if value is finite to avoid ValueErrors when turning it into
            # a hexagesimal string.
```

Reply:
✅